src/System.Management.Automation/engine/regex.cs

```diff
@@ -148,15 +148,8 @@ private void Init()
                 return;
```

Comment:
@iSazonov, your last commit had 527 failures in `PowerShell-CI-linux`
(These are 5 of the failures)

Clean up open Runspaces when exit powershell process.Error occurred in Describe block
```powershell
The type is not a known type for the file system. Only "file","directory" or "symboliclink" can be specified.
At /home/vsts/work/1/a/bins/publish/Modules/Pester/4.8.0/Functions/TestDrive.ps1:53 char:5
```
Experimental Feature Basic Tests - Feature-Enabled.Error occurred in Describe block
```powershell
The type is not a known type for the file system. Only "file","directory" or "symboliclink" can be specified.
At /home/vsts/work/1/a/bins/publish/Modules/Pester/4.8.0/Functions/TestDrive.ps1:53 char:5
```
Enable-ExperimentalFeature and Disable-ExperimentalFeature tests.Error occurred in Describe block
```powershell
The type is not a known type for the file system. Only "file","directory" or "symboliclink" can be specified.
At /home/vsts/work/1/a/bins/publish/Modules/Pester/4.8.0/Functions/TestDrive.ps1:53 char:5
```
Validate Update-Help from the Web for all PowerShell modules..Error occurred in Describe block
```powershell
The type is not a known type for the file system. Only "file","directory" or "symboliclink" can be specified.
At /home/vsts/work/1/a/bins/publish/Modules/Pester/4.8.0/Functions/TestDrive.ps1:53 char:5
```
Language Primitive Tests.Error occurred in Describe block
```powershell
The type is not a known type for the file system. Only "file","directory" or "symboliclink" can be specified.
At /home/vsts/work/1/a/bins/publish/Modules/Pester/4.8.0/Functions/TestDrive.ps1:53 char:5
```